kahuna/public/stylesheets/main.css

```diff
@@ -1430,12 +1431,22 @@ FIXME: what to do with touch devices
     overflow: hidden;
 }
 
+.image-info__group--fixed-panel .labeller{
```

Comment:
Could we not just add this as a state to the labeller e.g. `.labeller--single-row`?